channels-rke2.yaml

```diff
@@ -24,6 +24,8 @@ appDefaults:
         defaultVersion: '1.31.x'
       - appVersion: '>= 2.11.0-0 < 2.12.100-0'
         defaultVersion: '1.32.x'
+      - appVersion: '>= 2.12.0-0 < 2.13.100-0'
```

Comment:
@ShylajaDevadiga I see that the versions were updated for v1.32.x as well, but not v1.31. Is this intentional or not required?

Reply:
@ShylajaDevadiga 
Discussed this with @snasovich offline - it seems the confusion is around the use of `appDefaults`. This field defines the default Kubernetes version range for each Rancher server version range. We've already configured this correctly for 2.12, so I'm resolving this comment, no changes needed here. 
```
- appVersion: '>= 2.12.0-0 < 2.13.100-0'
  defaultVersion: '1.33.x'
  ```